driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -502,33 +526,45 @@ public void onSet(Connection connection, Message.Response response, long latency
                                 connection.release();
                                 // Try another node
                                 logger.warn("Host {} is overloaded, trying next host.", connection.address);
```

Comment:
Seems like logging "trying next host" is not really consistent anymore since the decision is taken later by the RetryPolicy.